tools/file_packager.py

```diff
@@ -1062,14 +1089,13 @@ def generate_js(data_target, data_files, metadata):
       if (!fetched) {
         // Note that we don't use await here because we want to execute the
         // the rest of this function immediately.
-        fetchRemotePackage(REMOTE_PACKAGE_NAME, REMOTE_PACKAGE_SIZE).then((data) => {
-          if (fetchedCallback) {
-            fetchedCallback(data);
-            fetchedCallback = null;
-          } else {
-            fetched = data;
-          }
-        })
+        let packageData = await fetchRemotePackage(REMOTE_PACKAGE_NAME, REMOTE_PACKAGE_SIZE);
```

Comment:
See the comment above, we need to avoid `async` here since we don't want to suspend the function at this point